lib/src/signed_video_h26x_common.c

```diff
@@ -1312,3 +1313,10 @@ signed_video_compare_versions(const char *version1, const char *version2)
 error:
   return status;
 }
+
+bool
+signed_video_is_golden_sei(signed_video_t *self, const uint8_t *sei, size_t sei_size)
+{
+  h26x_nalu_t nalu = parse_nalu_info(sei, sei_size, self->codec, false, true);
```

Comment:
You will get a segmentation fault here if `self` is a null pointer. Since this is a public API, there should be sanity checks on the input parameters.